extensions/vscode/.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "@typescript-eslint/naming-convention": [
+            "warn",
+            {
+                "selector": "import",
+                "format": [ "camelCase", "PascalCase" ]
+            }
+        ],
+        "@typescript-eslint/semi": "warn",
+        "curly": "warn",
+        "eqeqeq": "warn",
+        "no-throw-literal": "warn",
+        "semi": "off"
+    },
+    "ignorePatterns": [
+        "out",
+        "dist",
+        "**/*.d.ts"
+    ]
+}
```

Comment:
We gotta format this file and there's a newline at the end of this file missing. By formatting I mean sticking to our code convention that is using 2 spaces for indentation. What is this eslint config for? Cause we have an eslint plugin which we also use for TutorialKit itself so it'd be nice if we can use that here as well. But I need to understand better what this is for.

Reply:
This one is generated by the official vscode extension generator, I have no more insights as to its relevance;)

Reply:
Interesting. I think we can remove this because the extension lives in our monorepo which has eslint and enforces our code conventions so I'd say we delete this one.

Reply:
Which also means we can remove the `eslint` dev dependency from it.